### ISSUE TITLE
Harrison MixBus32C / Ardour

### ISSUE THREAD

**Issue by stevespaw** [bitfocus/companion-module-requests#470]
MixBus DAW based in similar code Ardour
OSC Control similar to other audio consoles (x32) and DAW's like Reaper.
[https://github.com/bitfocus/companion-module-behringer-x32](url)
[https://github.com/bitfocus/companion-module-cockos-reaper](url)

MixBus [https://harrisonconsoles.com/product/mixbus/](url)
Ardour [https://ardour.org/](url)

Mixbus OSC [http://www.harrisonconsoles.com/mixbus/mixbus6-live-manual/1/en/topic/osc-control](url)
Ardout OSC [http://manual.ardour.org/using-control-surfaces/controlling-ardour-with-osc/osc-setup-dialog/](url)
![image](https://user-images.githubusercontent.com/11045430/114187672-e0331200-990d-11eb-9a03-44dc705f48d4.png)

**Comment by josephdadams** [comment]
Is this the same as #324 ?

**Comment by stevespaw** [comment]
Yes, it is for the most part, The Mixbus versions will most likely update before Ardour.

**Comment by JTF4** [comment]
I am currently working on a module for this. What features would you like to see included @stevespaw?

**Comment by LGCW** [comment]
Basic transport (play/stop/rec) and mutes would be nice.

**Comment by stevespaw** [comment]
That is great,
We are using it mostly as a mixer now. 
Solo's
Mutes
being able to turn plug-ins on and off
Mixbus sends on and off
mastersend on-off
eq on-off
and as many of the monitor functions as possible.
![image](https://user-images.githubusercontent.com/11045430/119345166-247c3680-bc5e-11eb-901b-cde182e187ca.png)

in terms of feedback, it would be great to get:
some system status stuff
and, of course, the channel names
some db feedback, maybe like vMix. to get some basic signal indication.

  That would be a great start, thanks again....  Steve

**Comment by stevespaw** [comment]
I have been wondering is it would be possible to actually just Control/Send HotKeys without having to use Vicreo(I have not used it much),  and having to make sure Mixbus is in the foreground. That would be super powerful.
[MixbusDefaultHotKeys.pdf](https://github.com/bitfocus/companion-module-requests/files/6532264/MixbusDefaultHotKeys.pdf)

**Comment by oregondean** [comment]
Like @stevespaw, I'd like basic functions ... in my case even more basic:
I'd be thrilled with just channel fade and mute, and master fade and mute to start with.  I use the MixBus software to do initial setup.  Streamdeck is just to make basic production easier for basic functions so I don't have to make the App active and then mouse around to get stuff done.  And especially to quickly mute when feedback crops up!
